src/algorithms/sorting/short_bubble_sort.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var Comparator = require('../../util/comparator');
+
+/**
+ * short bubble sort algorithm
+ * worst: O(n^2) best: O(n)
+ */
+
+function shortBubbleSort(array, comparatorFn) {
+  var comparator = new Comparator(comparatorFn);
+  var length = array.length;
+  var i = 0;
+
+  for (i; i < length; i++) {
+    var current = array[i];
+    var next = array[i+1];
+
+    if (next === undefined) break;
+    if (comparator.lessThanOrEqual(current, next)) continue;
```

Comment:
I have a habit of breaking the execution using return; break; continue; But in this case really makes more sense to use as you said.